src/components/stringclass.js

```diff
@@ -0,0 +1,249 @@
+       if(name instanceof Array){
+        return "This is already an array";
+        } else{
+            if(name.includes(" ")){
+                return name.split(" ");
+            } else{
+                return name.split("");
+            }
+        } 
+    }
+    
+    
+}
+
+function wordCount(name){
+    /*Returns the number of words in the string */
+    if(wordCheck(name)=== true){
+        return name.split(" ").length;
+        
+    }
+}
+
+function toCurrency(name){
+    /*Returns a currency representation of the String */
+    if(wordCheck(name)=== true){
+        let currencyString = Number(name);
+        if(typeof currencyString === "number" && !isNaN(currencyString)){
+            return currencyString.toLocaleString();
+        } else{
+            return "This is not a number";
+        }
+    }
+}
+
+function fromCurrency(name){
+    /*Returns a number representation of the Currency String */
+    if(wordCheck(name)=== true){
+        let currencyString = name.replace(/\,/g,'');
+        if(!isNaN(Number(currencyString))){
+            return currencyString;
+        } else{
+            return "This is not a number"; 
+        }
+    }
+}
+
+function inverseCase(name){
+    /*Returns each letter in the string as an inverse of its current case */
+    if(wordCheck(name)=== true){
+        
+        let regexpUpper = /^[A-Z]/;
+        let splitString = name.split("");
+        let inverseString = splitString.map(function(word){
+            if(regexpUpper.test(word) === true){
+                return word.toLowerCase();
+                
+            } else{
+                return word.toUpperCase();
+            }
+        });
+        return inverseString.join("");
+    }
+
+}
+
+function alternatingCase(name){
+    /*Returns the letters in alternating cases */
+    if(wordCheck(name)=== true){
+        
+        let words = name.toLowerCase().split("");
+        for (let index = 0; index < words.length; index += 2) {
+            words[index] = words[index].toUpperCase();
+        }
+        return words.join("");
+    }
+}
+
+function getMiddle(name){
+    /*Returns the character(s) in the middle of the string */
+    if(wordCheck(name)=== true){
+        
+        let position;
+        let length;
+        if(name.length % 2 === 1) {
+            position = name.length / 2;
+            length = 1;
+        } else {
+            position = name.length / 2 - 1;
+            length = 2;
+        }
+        let result = name.substring(position, position + length);
+        return result;
+    }
+}
+
+function numberWords(name){
+    /*Returns the numbers in words */
+    if(wordCheck(name)=== true){
+        if(!isNaN(Number(name))){
+            let numbers = {0: "and",1:"one",2:"two",3:"three",4:"four",
+            5:"five",6:"six",7:"seven",8:"eight",9:"nine"};
+            let splitNumber = name.toLocaleString().split(",");
+            let listNums = [];
+            for(let i=0; i<splitNumber.length; i++){
```

Comment:
i'd change [line 197](https://github.com/wanjiku-carol/string-class/pull/6/files?diff=split#diff-db5888d39d0d4044481dcce4431e7c8cR197) to `let splitNumber = name.toString().split("");` and simplify this to 
```
 for(let i=0; i<splitNumber.length; i++){
       listNums.push(numbers[splitNumber[i]]);
}
```